src/configly/utilities.py

```diff
@@ -0,0 +1,32 @@
+def quote_string(value: str):
+    r"""Add syntactically-correct quotation characters to a string value.
+
+    >>> quote_string("foo")
+    '"foo"'
+
+    >>> quote_string("'bar'")
+    "'bar'"
+
+    >>> quote_string("ba'z")
+    '"ba\'z"'
+
+    >>> quote_string('''q"u'x''')
+    '\'q"u\'x\''
+    """
+    if value.startswith('"') and value.endswith('"'):
+        return value
+
+    if value.startswith("'") and value.endswith("'"):
+        return value
+
+    if '"' not in value:
+        return f'"{value}"'
+
+    if "'" not in value:
+        return f"'{value}'"
+
+    # fmt: off
```

Comment:
lol never seen this one before, good to know

Reply:
yeah - it came in handy here, black kept on changing `"\'"` to `"'"` because they are == in python but this is one of those usecases where the explicit escaping is intended.